.github/ISSUE_TEMPLATE/releases/release_checklist.md

```diff
@@ -2,151 +2,254 @@
+            `eng/Versions.props` matches the previous version of the
+            release artifacts.
+        - These can be found in the `dotnetcli` storage account in the `source-build-artifacts`
+          blob container.  The access key for this storage account
+          is in Azure Keyvault in the `EngKeyVault` vault in the
+          "Dotnet Engineering Services" subscription.  You can access
+          the key vault with [VaultExplorer](https://aka.ms/ve) and
+          the storage account with [Azure Storage Explorer](https://azure.microsoft.com/en-us/features/storage-explorer/).
+          Use the latest version of the artifacts in the branch we
+          are building, which should be from the latest released
+          version.  For example, when building 3.1.419, use
+          `Private.SourceBuilt.Artifacts.0.1.0-3.1.418-1728976-20220420.1.tar.gz`.
+          The version number is `0.1.0-3.1.418-1728976-20220420.1`.
+      - [ ] In `global.json`, update `dotnet` to the latest released 3.1 SDK
+            version. It should be one version behind the version we are building.
+            For example, if we are building 3.1.214, then the line should read
+            `"dotnet": "3.1.213"`.
+      - [ ] Update feeds:
+        - [ ] If new feeds were added to NuGet.config (i.e. [there's a diff
+              in NuGet.config](https://github.com/dotnet/source-build/commit/f07ff3ab98992e460ae605f609783f3abba20601#diff-9c5952957709545aad811b400e7e516eebade1e44fc4fbc23203403ffeb92c34)),
+              copy them to [smoke-testNuGet.config](https://github.com/dotnet/source-build/blob/release/3.1/smoke-testNuGet.Config)
+              and the `VSS_NUGET_EXTERNAL_FEED_ENDPOINTS` section in `build.sh`.
+        - [ ] Add the servicing feed to build.sh and smoke-testNuGet.config
+          1. The URL will look like this: <https://pkgs.dev.azure.com/dnceng/internal/_packaging/3.1.419-servicing.22219.28-shipping/nuget/v3/index.json>
+          1. In build.sh, add this feed to the `VSS_NUGET_EXTERNAL_FEED_ENDPOINTS`.
+          1. In smoke-testNuGet.config, add another feed entry that looks like
+             the others, with key="5.0.408-servicing.22219.28-shipping"
+             (for example) and value as the servicing url.
+      - [ ] In `eng/Version.Details.Xml`, verify that the `<Sha>` tag for all
+            repos were updated, except for these expected to usually be stable:
+        - arcade
+        - sourcelink
+        - aspnet-xdt
+        - cliCommandLineParser
+        - xliff-tasks
+        - Newtonsoft.Json
+        - common
+        - Roslyn (except for feature band updates)
+        - MSBuild (except for feature band updates)
+        - NuGet-client (except for feature band updates)
+        - fsharp (except for feature band updates)
+        - VSTest (except for feature band updates)
+1. - [ ] [Regenerate patches as necessary](patch-updates.md).
+      - Run `./build.sh` to begin clone and patch application and see what,
```

Comment:
This step will probably fail if we don't have the PAT setup from the steps below. Consider moving this after "Start building locally..." and before "Review prebuilt baseline diff...".

Reply:
Good point, changed as suggested.